frost/bip340.go

```diff
@@ -80,17 +83,13 @@ func (bc *Bip340Curve) IsPointOnCurve(p *Point) bool {
 
 // SerializedPointLength returns the byte length of a serialized curve point.
 func (b *Bip340Curve) SerializedPointLength() int {
-	// From the Marshal() function of secp256k1 go-ethereum implementation:
-	// 	 byteLen := (BitCurve.BitSize + 7) >> 3
-	//   ret := make([]byte, 1+2*byteLen)
-	return 65
+	byteLen := (b.BitSize + 7) >> 3
```

Comment:
Non-blocking: I would take an opportunity an explain the bitwise magic for future-us. For example, here we divide by `8` but it took me a while to figure this out 😅 Same regarding other places in-scope of this PR.